test/fillholes.jl

```diff
@@ -0,0 +1,64 @@
+@testset "fillhole" begin
+    #binary
+    img = Bool[
+        0 0 0 0 0 0 0
+        0 1 1 1 1 1 0
+        0 1 0 0 0 1 0
+        0 1 0 0 0 1 0
+        0 1 0 0 0 1 0
+        0 1 1 1 1 1 0
+        0 0 0 0 0 0 0
```

Comment:
Could you add a test where the 0s at the border are not a single connected component? Also the case where there are no zeros on the border m. Also add a test for  multiple interior holes.

Reply:
Fixed

Reply:
Note that we could say nothing for "holes" in the border, anyway we don't have acess to underlyting image domain, are these really holes
So as other image processing framework we don't fill hole touching the borders